movie_REST_API/movie_REST_API/App/Controllers/InfoMovieViewController.swift

```diff
@@ -91,21 +98,19 @@ final class InfoMovieViewController: UIViewController {
         action()
         loadMoviesData()
     }
-    
-    // MARK: - Public Method
-    func createPresentImage(image: String?) {
-        
-        guard let imageString = image else { return }
-        let urlString = Constant.firstPartURLString + imageString
-        guard let imageURL = URL(string: urlString) else { return }
-        getImageDataFrom(url: imageURL)
+
+    // MARK: - Public Methods
+
+    func createPresentImage() {
+        presenter?.getImageDataFrom()
     }
-    
-    // MARK: - Private Method
+
+    // MARK: - Private Methods
+
     private func action() {
```

Comment:
```suggestion
    private func action() {
```
action что за название?